remote/common_test.go

```diff
@@ -0,0 +1,202 @@
+package remote
```

Comment:
Code is borrowed from actor/common_test.go and router/common_test.go to follow the same testing strategy.